db/checkEnvVariables.ts

```diff
@@ -0,0 +1,48 @@
+
+import fs from 'fs'
+import os from 'os'
+
+const tempPath = os.tmpdir()
+
+export const getEnvParametersFromEnv = () => { 
+	if(process.env.token && process.env.DATABASE_URL) 
+		return { token: process.env.token, databaseUrl: process.env.DATABASE_URL }
+}
+
```

Comment:
This is undocumented behaviour, right? You mention the `.env` file, but use some other file here...

Reply:
Actually this is a function (setenv) that sets another file as environment parameters so the user can call the main function in any path they want.